packages/integrations/cloudflare/src/index.ts

```diff
@@ -67,8 +67,8 @@ export default function createIntegration(args?: Options): AstroIntegration {
 					}
 
 					vite.ssr = {
-						target: 'webworker',
-						noExternal: true,
```

Comment:
We were using noExternal to force everything into a single bundle because that's a requirement of Cloudflare. Is this not needed now because of the second build by esbuild?

Reply:
Yes the bundle option of esbuild is performing that single file build, `noExternal: true`was also creating errors.